src/tooltip-menus/items.js

```diff
@@ -6,12 +6,9 @@ import {
   isFirstRowSelected,
   enableCellsColor,
 } from './utils';
-import {
-  getDeleteCommand,
-  changeCellsBackgroundColor,
-  toggleTableHeaders,
-} from './commands';
+import {changeCellsBackgroundColor, toggleTableHeaders} from './commands';
 import {createElementWithClass} from '../util';
+import {getDeleteCommand} from '../input';
```

Comment:
nit - spaces
might want to lint at this point, this library is becoming very complex :)

Reply:
sounds good, adding eslint to the bugs cycle branch is ok? or we want it in this pr already?

Reply:
yes - can be a new ticket!